readme.md

```diff
@@ -17,11 +17,21 @@ Example exclude file:
 .env
 ```
 
-Then execute the build script:
+Next, configure the username/password in badtrack_secrets.py (Hint: Ctrl+F search 'export')
+
+Next, build badtracksecrets:
+
+`python3 ./badtrack_secrets.py`
+
+Then, install it:
+
+`sudo dpkg -i badtracksecrets.deb`
+
+Then build badtrack:
 
 `./build_badtrack.py`
 
-Finally install the package:
+Finally, install badtrack:
```

Comment:
What if you do:
```
dpkg -i badtracksecrets.deb badtrack.deb
```
Also what happens if you keep running above repeatedly, will dpkg just skip and say it's already installed?

Reply:
it reinstalls both packages